components/dashboard/src/admin/License.tsx

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { PageWithSubMenu } from "../components/PageWithSubMenu";
+import { adminMenu } from "./admin-menu";
+
+import { LicenseContext } from "../license-context";
+import { ReactElement, useContext, useEffect } from "react";
+import { getGitpodService } from "../service/service";
+
+import { ReactComponent as Alert } from "../images/exclamation.svg";
+import { ReactComponent as Success } from "../images/tick.svg";
+import { ReactComponent as Tick } from "../images/check.svg";
```

Comment:
Ah okay, I assumed we use icons. Now that you mention it, I think it might make it much simpler to use HTML codes for check and cross. I have changed the code to reflect this. LMK how the new changes look in design https://gitpod.nanpod.fun/admin/license and https://nvn-license-dash.staging.gitpod-dev.com/admin/license

Reply:
> I assumed we use icons.

Most of the times we go with a custom icon. ✔️ 

**issue**: Looks good overall on macOS but I'm concerned that going with this could slightly affect UX as rendering could look quite different on different browsers and operating systems. Do you know how this will render in Linux a system?

**suggestion**: Sounds safer to go with a custom icon taking also into consideration the increased complexity for supporting different icons for light and dark theme. Maybe we could also simply use the `dark:filter-invert` class which we're using elsewhere. Sorry for the back and forth, @nandajavarma! What do you think?

Reply:
agree! I have added the icons back! changed the `Check` icon to one you suggested. It should be [pretty uniform now.](https://nvn-license-dash.staging.gitpod-dev.com/admin/license)